requirements-dev.txt

```diff
@@ -8,12 +8,14 @@ splinter==0.17.0
 selenium==4.1.2
 twine==3.8.0
 Pallets-Sphinx-Themes==2.0.2
+docutils>=0.17
 Sphinx==4.4.0
 sphinx-issues==3.0.1
-sphinx-jinja==1.4.0
-sphinx-tabs==3.2.0
+sphinx-jinja @ git+http://github.com/return42/sphinx-jinja.git
```

Comment:
Interim solution / see https://github.com/tardyp/sphinx-jinja/pull/30 and https://github.com/tardyp/sphinx-jinja/issues/28#issuecomment-1060701874